projects/l2_pooling/multi_column_convergence.py

```diff
@@ -275,17 +356,37 @@ def runExperimentPool(numObjects,
   return result
 
 
-def plotConvergenceStats(convergence, columnRange, featureRange):
+def plotConvergenceByColumn(results, columnRange, featureRange):
   """
-  Plots the convergence graph
+  Plots the convergence graph: iterations vs number of columns.
+  Each curve shows the convergence for a given number of unique features.
+  """
+  ########################################################################
+  #
+  # Accumulate all the results per column in a convergence array.
+  #
+  # Convergence[f,c] = how long it took it to converge with f unique features
+  # and c columns.
+
+  convergence = numpy.zeros((max(featureRange), max(columnRange) + 1))
+  for r in results:
+    convergence[r["numFeatures"] - 1,
+                r["numColumns"]] += r["convergencePoint"]
 
-  Convergence[f,c] = how long it took it to converge with f unique features
-  and c columns.
+  convergence /= numTrials
 
-  Features: the list of features we want to plot
-  """
+  # For each column, print convergence as fct of number of unique features
+  for c in range(2, max(columnRange) + 1):
+    print c, convergence[:, c]
```

Comment:
nit: now that we run the experiment with 1-column, we can print this for c=1 as well.